SU2_CFD/include/drivers/CDriver.hpp

```diff
@@ -707,6 +732,111 @@ class CDriver {
    */
   map<string, string> GetAllBoundaryMarkersType();
 
+  /*!
+   * \brief Set the mesh displacement for the elasticity mesh solver.
+   * \param[in] iMarker - Marker identifier.
+   * \param[in] iVertex - Vertex identifier.
+   * \param[in] DispX - Value of the mesh displacement in the direction X.
+   * \param[in] DispY - Value of the mesh displacement in the direction Y.
+   * \param[in] DispZ - Value of the mesh displacement in the direction Z.
+   */
+  void SetMeshDisplacement(unsigned short iMarker, unsigned long iVertex, passivedouble DispX, passivedouble DispY, passivedouble DispZ);
+
+  /*!
+   * \brief Communicate the boundary mesh displacements in a python call
+   */
+  void CommunicateMeshDisplacement(void);
+
+  /*!
+   * \brief Return the sensitivities of the mesh boundary vertices.
+   * \param[in] iMarker - Marker identifier.
+   * \param[in] iVertex - Vertex identifier.
+   * \return Vector of sensitivities.
+   */
+  vector<passivedouble> GetMeshDisp_Sensitivity(unsigned short iMarker, unsigned short iVertex);
+
+  /*!
+   * \brief Set the load in X direction for the structural solver.
+   * \param[in] iMarker - Marker identifier.
+   * \param[in] iVertex - Vertex identifier.
+   * \param[in] LoadX - Value of the load in the direction X.
+   * \param[in] LoadX - Value of the load in the direction Y.
+   * \param[in] LoadX - Value of the load in the direction Z.
+   */
+  void SetLoads(unsigned short iMarker, unsigned short iVertex, passivedouble LoadX, passivedouble LoadY, passivedouble LoadZ);
+
+  /*!
+   * \brief Return the displacements from the FEA solver.
+   * \param[in] iMarker - Marker identifier.
+   * \param[in] iVertex - Vertex identifier.
+   * \return Vector of displacements.
+   */
+  vector<passivedouble> GetDisplacements(unsigned short iMarker, unsigned short iVertex);
```

Comment:
For methods of CDriver I think the names of these functions should be more descriptive.
Also they are not implemented yet but I am a bit concerned about returning vector by value, depending on the use case it may either be perfectly fine or not very efficient, can you elaborate on how this is going to be used?